README.md

```diff
@@ -107,6 +107,19 @@ optional arguments:
 
 The script will generate a HTML file named `heatmap.html`. This file will automatically open in your browser once the script completes. Enjoy!
 
+## Running with Docker
+
+With Docker this script can be executed without the need for installing any dependencies. 
+
+Assume that the Google takeout archive is located here: `/some/path/takeout.zip`. Run the following
+Docker command to generate a `heatmap.html` file under `/some/path/`:
+
+```bash
+$ docker run --rm -v /some/path/:/data melkamar/geo-heatmapa /data/takeout.zip
```

Comment:
``melkamar`` is probably not the desired username of the Docker image.

Also i would be better to suggest building the Dockerimage locally unless luka sets up the building himself. We handle quite personally data here and allowing your image to be used is a major attack vector and shouldn't be trusted.

Reply:
Yeah, as mentioned in the comment, the idea is that @luka1199 can create their own image in Docker Hub and then change the name of the image. I cannot publish the Docker image in their namespace.

I wanted to make it as quick as possible to use, but building the image locally could be a reasonable compromise between speed and trust. I'll add that to the readme, thanks.

Reply:
Suggestion to use github package repository https://help.github.com/en/github/managing-packages-with-github-packages/about-github-packages#supported-clients-and-formats to host the docker image, therefore there wouldn't be a namespace issue as it is per repository